### PR TITLE
Some New Rules !

### PR DESCRIPTION
**Pull request by ghost**
2 More new rules Have a look On it

### CODE REVIEW

**Comment by RussellLVP** [comment]
Thank you for the change!

There are two things I think are important to point out. Firstly, I'd prefer it if we wouldn't remove the line informing people how to contact in-game crew, as this is actually being used quite a lot.

Secondly, the two rules that you're adding are redundant with each other. I would consider advertising to be a form of spam. Additionally, I think it's OK to name a different server once or twice - people are free to go wherever they want to.

Would you mind updating the PR to keep the "If you have questions.." line, and only adding the "Do not spam!" line above there?

**Comment by ghost** [comment]
Thx For Comment  ! 

just add New rules Donot Spam ! Plzz

File: data/commands/rules.json [hunk 1]
```diff
@@ -5,5 +5,6 @@
     "{2566BA}- {A9C4E4}The Ship is a peace zone; don't harm players on it and don't run to it if you're fighting!",
     "{2566BA}- {A9C4E4}Do not abuse bugs or commands.",
     "{2566BA}- {A9C4E4}Do not harm a cruise in any way!",
-    "If you have questions, use {D67513}@<message> {A9C4E4}to contact the Las Venturas Playground staff. Have fun!"
```
Comment (RussellLVP) [inline data/commands/rules.json:8]: Please keep this line (as the last one in the list)

File: data/commands/rules.json [hunk 2]
```diff
@@ -5,5 +5,6 @@
...
+    "{2566BA}- {A9C4E4}Advertising of Server/Website,Is Not Allowed !",
```
Comment (RussellLVP) [inline data/commands/rules.json:8]: We don't need this line (it's redundant with line 9).

**Comment by ghost** [comment]
ty xD

**Comment by ghost** [comment]
<3

File: data/commands/rules.json (hunk 2 above)
Comment (ghost) [inline data/commands/rules.json:8]: ok

Comment (ghost) [inline data/commands/rules.json:8]: can u do it plzz

Comment (RussellLVP) [inline data/commands/rules.json:8]: I'm more than happy to help you make the change. Do you not know how to update the PR? What client are you using?

**Comment by ghost** [comment]
Bro what is PR !

**Comment by RussellLVP** [comment]
This is called a "pull request" - you're proposing a change to Las Venturas Playground. Just like you made this pull request, you can update it as well.

Since I left some comments on the change, the idea is that you'd update your proposal, I (or another developer) takes another look and then we _accept_ the change, which means that it'll be included in a future Las Venturas Playground version.

**Comment by ghost** [comment]
https://github.com/HopperLVP/playground/blob/34bb1a8b7c8f69ce4413a13bed604c665b21cb9b/data/commands/rules.json

**Comment by RussellLVP** [comment]
Almost! You removed the wrong rule! :)

It's the "{2566BA}- {A9C4E4}Do not spam!", rule that we want to add.

**Comment by ghost** [comment]
ok 
i will edit again]
wait 2 min
xD

**Comment by ghost** [comment]
https://github.com/HopperLVP/playground/blob/master/data/commands/rules.json

**Comment by ghost** [comment]
now it Ok russell

**Comment by RussellLVP** [comment]
Perfect! Now make a new pull request with the latest change :)

There should be a button on https://github.com/HopperLVP/playground

**Pull request closed without merging**